www/pad/export.js

```diff
@@ -96,7 +96,34 @@ define([
             if (ext === ".md") {
                 var md = Turndown({
                     headingStyle: 'atx'
-                }).turndown(toExport);
+                }).addRule('table', {
+                filter: ['tr'],
+                replacement: function (content, node) {
+                    var indexOf = Array.prototype.indexOf;
+                    var index = indexOf.call(node.parentNode.childNodes, node);
+                    var rowContent = node.innerHTML.replace(/<td>/g, '').replace(/<br>/g, ' ').split('</td>');
```

Comment:
1. Removing the `<br>` here will remove new lines inside cells. This is acceptable since currently exports are not working but it would be nice to be able to fix this.
2. This won't work if the table contains "header" cells which are `<th>` instead of `<td>`
3. I think it would be easier to work with the DOM elements directly instead of treating the HTML as a string